readme.md

```diff
@@ -3,6 +3,7 @@
 # h3-py
 
 [![PyPI version](https://badge.fury.io/py/h3.svg)](https://badge.fury.io/py/h3)
+[![PyPI downloads](https://pypip.in/d/h3/badge.png)](https://crate.io/packages/h3/)
```

Comment:
Actually, this link isn't working for me. Is https://crate.io/packages/h3/ the right link?

Reply:
good catch. fixed!